Dockerfile.tests

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM ubuntu:latest
+FROM ubuntu:22.04
```

Comment:
Do we need that change?

Reply:
It did not work without it, let's leave it as it right now and then we can debug it.